scripts/smart-dump.ts

```diff
@@ -34,37 +38,51 @@ program.command('dump [recipe] [env]').action(async (recipe, env) => {
     recipe = './smart-dump/defaultRecipe.js';
   }
 
+  const tempDumpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'oc-export-'));
+  logger.info(`>>> Temp directory: ${tempDumpDir}`);
+
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const { entries, defaultDependencies } = require(recipe);
   const parsed = {};
   const date = new Date().toISOString().substring(0, 10);
   const hash = md5(JSON.stringify({ entries, defaultDependencies, date })).slice(0, 5);
-  const filename = `${date}.${hash}`;
-  let docs = [];
+  const seenModelRecords: Set<string> = new Set();
+
+  const gitRevision = execSync('git describe --always --abbrev=0 --match "NOT A TAG" --dirty="*"').toString().trim();
```

Comment:
That could fail in production, where (I believe) `git` is not available. Not a blocker, but we should ideally implement a fallback.